aml-development/frontend
# fix(listing): fix various bugs discovered on listing page

**Description**
Fix for 3 issues:
1. Changing the listing ID in the URL does not reload the reviews or notifications (administration)
2. Clicking on 'See More' in Listing Overview resets the carousel to the first image
3. Submitting a review does not refresh review stats in listing header and left-hand section

**Acceptance Criteria**
1.  When viewing a listing, changing the listing ID in the URL fully refreshes the listing page
2.  When using the "See More/Less" functionality in the Listing Overview section, the carousel image remains the same
3.  When submitting a review for a listing, the listing page is properly updated to reflect the new review

**How to Test**
Pull this branch

Navigate to any listing.  Change the listing ID in the URL to a different listing.  Ensure that each section (Overview, Details, Reviews, Administration) has updated accordingly.

Navigate to a listing with multiple screenshots (ie Baltimore Ravens).  Ensure that clicking on 'See More' in overview does not reset the carousel to the first image if it's not already on the first image.

Submit a review on a listing.  Ensure the listing page is reloaded with the correct information.
